tasks/install.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Download Amazon Inspector Agent Installer
+  become: yes
+  get_url:
+    url: "{{ sansible_amazon_inspector_agent_url }}"
+    dest: "{{ sansible_amazon_inspector_agent_dest }}"
+    mode: 0755
```

Comment:
Should we have a checksum for the install script? That way we are a bit safer against someone hijacking the script.

Reply:
GPG solution implemented.